README.md

```diff
@@ -1 +1,35 @@
-# nppThemes
\ No newline at end of file
+# Notepad++ Themes Collection
+
+Notepad++ ships with around 20 [themes](https://npp-user-manual.org/docs/themes/) available to define the formatting rules for normal text and syntax highlighting. However, users can create their own themes, or get themes from somewhere else.
+
+To facilitate sharing of user-created themes, the Notepad++ Developers have create this Themes Collection to house the XML files for the user-created themes.
+
+## Using a Theme from this Collection
+
+For now, you have to manually download and install a Theme from this Collection.
+
+1. Download the XML file from the [`themes/` folder](./themes) of this Collection.
+   - From the [`themes/` folder](./themes), click on the name of the theme's XML file.
```

Comment:
Those links currently _appear_ to not work, because there is no `themes/` folder yet, because there are no themes published yet.  I verified (in a temporary branch in my fork) that as soon as there is a themes/ folder, the link will properly resolve.

Reply:
Did you want me to grab the ["dark side of ruby blue"](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/12038), so that we can create/populate the themes folder, so that the links will work right away?  If so, should it be in this PR or a separate PR?

Reply:
> Those links currently appear to not work, because there is no themes/ folder yet, because there are no themes published yet.

Please add "themes" folder plus a 0 length file named "dummy".

> Did you want me to grab the https://github.com/notepad-plus-plus/notepad-plus-plus/pull/12038, so that we can create/populate the themes folder, so that the links will work right away? If so, should it be in this PR or a separate PR?

I would let @airduv do it himself - let's keep the author's credit - it's important for both project and contributor.
We  will see another solution if there's no response from him.